examples/in_depth_component_example.py

```diff
@@ -125,7 +123,7 @@ async def handle_responses(
             and e.interaction.message == message
         )
     ) as stream:
-        async for event in stream:
+        for event in stream:
```

Comment:
Are you sure this isn't an async for? If not where is the waiting done?

Reply:
You are correct here.